smarts/core/utils/math.py

```diff
@@ -307,9 +307,6 @@ def position_at_offset(
     if is_close(dist, offset):
         return p2
 
-    if offset > dist:
-        return None
-
     return p1[0] + (p2[0] - p1[0]) * (offset / dist), p1[1] + (p2[1] - p1[1]) * (
```

Comment:
Why was this removed?

Reply:
This case never gets hit, I looked through the code and found that if this case would get hit it would cause errors in the code since we always unpack the result without checking checking the truthy-ness.